integrations/src/main/java/com/oms/integrations/EmailHttpClientImpl.java

```diff
@@ -1,16 +1,62 @@
+
 package com.oms.integrations;
 
 import com.oms.dto.EmailRequestDto;
 import org.springframework.stereotype.Component;
-
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.MailException;
+import jakarta.mail.MessagingException;
+import java.util.Properties;
 import java.util.concurrent.atomic.AtomicInteger;
 
 @Component
 public class EmailHttpClientImpl implements EmailHttpClient {
     private AtomicInteger useCnt = new AtomicInteger();
 
+    @Autowired
+    private JavaMailSender javaMailSender;
+
+    @Value("${spring.mail.host}")
+    private String smtpHost;
+
+    @Value("${spring.mail.port}")
+    private int smtpPort;
+
+    @Value("${spring.mail.username}")
+    private String smtpUsername;
+
+    @Value("${spring.mail.password}")
+    private String smtpPassword;
+
     @Override
     public String sendEmail(EmailRequestDto emailRequestDto) {
-        return String.format("SUCCESS %d", useCnt.incrementAndGet());
+        try {
+            JavaMailSenderImpl mailSender = (JavaMailSenderImpl) javaMailSender;
+            mailSender.setHost(smtpHost);
+            mailSender.setPort(smtpPort);
+            mailSender.setUsername(smtpUsername);
+            mailSender.setPassword(smtpPassword);
+
+            Properties props = mailSender.getJavaMailProperties();
+            props.put("mail.transport.protocol", "smtp");
+            props.put("mail.smtp.auth", "true");
+            props.put("mail.smtp.starttls.enable", "true");
+            props.put("mail.debug", "true");
+
```

Comment:
There is no need to reconfigure the mail sender every time we send mail - a better option would be to configure it once (say first send)